addons/common/XEH_preInit.sqf

```diff
@@ -215,6 +215,7 @@ PREP(getConfigGunner);
 PREP(getConfigCommander);
 PREP(getHitPoints);
 PREP(getHitPointsWithSelections);
+PREP(getSelectionsWithoutHitPoints);
```

Comment:
A better name might be `getSelections` so it would mirror `getHitPoints`, that automatically implies it's without anything else.